Seal_of_Approval.md

```diff
@@ -0,0 +1,21 @@
+# Seal of Approval
+
+This is a list of packages in the `data.table` community.
+
+Further detail about these packages and their relationship to `data.table` can be found at [The Raft blog](https://rdatatable-community.github.io/The-Raft/#category=seal%20of%20approval).
+
+To add your package to this list, please [submit a Pull Request to The Raft](https://github.com/rdatatable-community/The-Raft/), making sure to follow the templated instructions.
+
+## Extension packages: Add to the internal functionality of `data.table`
+
+- [nc](https://github.com/tdhock/nc): Named capture regular expressions for text parsing and data reshaping.
```

Comment:
Maybe 'Named capture groups in regular expressions ... '?

Reply:
this was copied from https://github.com/tdhock/nc